birdhouse/components/weaver/docker-compose-extra.yml

```diff
@@ -114,6 +114,19 @@ services:
       timeout: 10s
       retries: 2
 
+  # Dedicated database for Weaver, since other 'mongodb' image does not employ the same version.
+  weaver-mongodb:
+    image: mongo:5.0
+    container_name: weaver-mongodb
+    networks:
+      - mongodb
```

Comment:
I think this is the same network name as the other mongodb !  Better not clash with the other one.

Reply:
I'm not sure if it would clash or not, but I'll separate it just to be sure.